lib/signaling/v2/room.js

```diff
@@ -665,12 +665,21 @@ function periodicallyPublishStats(roomV2, transport, intervalMs) {
         // StatsReport will add zeros to properties that do not exist.
         const report = new StatsReport(id, response, true /* prepareForInsights */);
 
-        transport.publishEvent('quality', 'stats-report', {
-          audioTrackStats: report.remoteAudioTrackStats,
-          localAudioTrackStats: report.localAudioTrackStats,
-          localVideoTrackStats: report.localVideoTrackStats,
-          peerConnectionId: report.peerConnectionId,
-          videoTrackStats: report.remoteVideoTrackStats
+        transport.eventObserver.emit('event', {
+          group: 'quality',
+          level: 'info',
+          name: 'stats-report',
+          payload: {
+            audioTrackStats: report.remoteAudioTrackStats.map((trackStat, i) =>
+              addAVSyncMetricsToRemoteTrackStats(trackStat, response.remoteAudioTrackStats[i])),
```

Comment:
instead of adding the metrics to status report, why not generate them as part of `StatusReport` ?

Reply:
I think we already generate different status report based on `prepareForInsights`

Reply:
@makarandp0 No, we just set different values, but the fields are the same. I don't want to conditionally add properties to the class.